go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -367,6 +469,37 @@ go.app = (function () {
           "User-Agent": ["Jsbox/Covid19-Triage-USSD"]
         }
       }).then(function () {
+        return self.states.create("state_submit_sms");
+      }, function (e) {
+        // Go to error state after 3 failed HTTP requests
+        opts.http_error_count = _.get(opts, "http_error_count", 0) + 1;
```

Comment:
@rudigiesler I'm assuming that even though the USSD session itself might time out, we'll still be trying to send the SMS 3 times?

Reply:
Yes, the worker will attempt to send up to 3 times (for any HTTP errors), and then send a response to the user. So it is possible that if a timeout happens here, we'll send the SMS, and then try to respond to the user's USSD session, but fail because the session has since timed out.

These retries are immediate though, so hopefully this is a short period of time. But there is a small chance that the timeout will happen while we're doing this, yes.